Sources/MassTransit/MassTransitUtils.swift

```diff
@@ -0,0 +1,22 @@
+import AsyncAlgorithms
+
+func randomString(length: Int) -> String {
+    let letters = "abcdefghijklmnopqrstuvwxyz0123456789"
+    var randomString = ""
+    for _ in 0..<length {
+        let randomIndex = Int.random(in: 0..<letters.count)
+        let letter = letters[letters.index(letters.startIndex, offsetBy: randomIndex)]
```

Comment:
`letters.randomElement()!` works as well

Reply:
It does! Nice, thanks!